frontend/lib/src/components/shared/StreamlitMarkdown/styled-components.ts

```diff
@@ -91,12 +91,18 @@ export const StyledStreamlitMarkdown =
         table: {
           // Add some space below the markdown tables
           marginBottom: theme.spacing.lg,
+          // Prevent double borders
+          borderCollapse: "collapse",
         },
 
         tr: {
           borderTop: `${theme.sizes.borderWidth} solid ${theme.colors.borderColor}`,
         },
 
+        th: {
+          textAlign: "inherit",
```

Comment:
if this has to be explicitly set, do we then also have to set `borderColor: "inherit"` as defined [here](https://github.com/streamlit/streamlit/pull/9794/files#diff-090003502eace910ce656401bc887a6e29d6e9a23a14e45a7979ff0854334d87L353) explicitly?

Reply:
I think this `borderColor: "inherit"` isn't required since we explicitly overwrite the borders for where we need it (see `th`, `td` below). I will play around with a few more advanced markdown tables, but I think the current set of options covers everything possible with markdown tables.

Reply:
I tried a couple of markdown table exampels, e.g. from https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/organizing-information-with-tables, and it seems to all work as expected.